add SUPPORT.md (helps point users from GitHub to appropriate support resources) (#6171)
scikit-image/scikit-image: .github/SUPPORT.md

## Support for scikit-image

Welcome to scikit-image! We use GitHub for tracking bugs and feature requests.

> GitHub [Issues](https://github.com/scikit-image/scikit-image/discussions)

Reviewer: https://github.com/scikit-image/scikit-image/issues :slightly_smiling_face:

> GitHub [Issues](https://github.com/scikit-image/scikit-image/issues)
> should be used for reporting bugs or performance regressions. We prefer use of
> GitHub [Discussions](https://github.com/scikit-image/scikit-image/discussions)
> rather than issues for feature requests. If you need support for using

Reviewer 2: ```suggestion
When you report an [issue](https://github.com/scikit-image/scikit-image/issues),
you may apply a label to it to specify its type: bug, feature request, performance,
infrastructure, documentation, etc.
If you need support for using
```
Right? I guess we can mark this draft PR as 'ready for review' :slightly_smiling_face:

> rather than issues for feature requests. This GitHub repository isn't the right

Reviewer: What if we removed this:
> This GitHub repository isn't the right place to get support for using scikit-image, but the following resources are
available for that purpose.
...

> scikit-image, please feel free to reach out to the developers and the community

Reviewer 2: either "please reach out" or "feel free to reach out"

> scikit-image, please feel free to reach out to the developers and the community

Author: ```suggestion
scikit-image, please reach out to the developers and the community
```

available for that purpose.

### Support

> 

Reviewer: ... and added this sentence here:
> If you need support for using scikit-image, please feel free to reach out to the developers and the community using the following resources.

[image.sc forum](https://forum.image.sc/tag/scikit-image): This is the
recommended place to post questions on how to perform a particular image
analysis task. The image.sc forum involves a larger image analysis community
across multiple software packages and thus has a better chance of finding
someone who has worked on a similar problem.

### Real-time Chat

> For chat, we have the [skimage zulip](https://napari.zulipchat.com). Feel free

Reviewer: https://skimage.zulipchat.com/ :wink:

to ask questions there or just say hello and let us know how you are using
scikit-image!

### Developer Discussion

> Discussions related to scikit-image development can also be raised with the scikit-image tag on the [Scientific Python discourse site](https://discuss.scientific-python.org/c/contributor/skimage).

Reviewer 2: ```suggestion
Discussions related to scikit-image development can also be raised with the
scikit-image tag on the [Scientific Python Discourse forum](https://discuss.scientific-python.org/c/contributor/skimage).
```
